manifest.json

```diff
@@ -12,19 +12,19 @@
     },
```

Comment:
Signature for `...caUq`:

`+pdUi99GEI/IU7dEtVCf2qByqdzZDYPbmOWknkEcndXsbyAAwrJW1qgdNQtf3kE+Vq5MqVCtPgffUERmJBuJAA==`